README.md

```diff
@@ -25,25 +25,25 @@ Usage
 You can use the lower level nvml bindings
 
 ```python
->>> from pynvml import *
->>> nvmlInit()
+>>> from pynvml.nvml import *
```

Comment:
It's a bit ugly, but you probably want to add 

```python
from .nvml import *
```

In `pynvml/__init__.py`

Then you can drop the `.nvml` 

(also, just as an FYI, using `import *` is considered a bit odd in Python these days, though I'm not sure how better to handle a library with such a large namespace like this.  Probably one should just `import pynvml` and use `pynvml.foo` everywhere, but that's a topic for another PR I think).